index.html

```diff
@@ -0,0 +1,281 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Hello Microverse act </title>
+  <link rel="stylesheet" href="./css/styles.css" />
+   <!-- Hamburger icon  Section-->
+  <script
+  src="https://kit.fontawesome.com/716817a6e7.js"
+  crossorigin="anonymous"
+></script>
+</head>
+
+<body>
+ <!-- Header Section -->
+ <header class="header">
+  <div class="header-container">
+    <a href="#" class="logo animation-rotate">kayjee</a>
+    <nav class="nav-bar">
+      <ul class="navigation-bar nav-menu">
+        <li><a href="#portfolio" class="list-item">Portfolio</a></li>
+        <li><a href="#about" class="list-item">About</a></li>
+        <li><a href="#contact" class="list-item">Contact</a></li>
+      </ul>
+    </nav>
+    <div class="hamburger">
+      <i class="menu-icon fa-solid fa-bars"></i>
+      <i class="cross-icon fa-solid fa-times"></i>
+    </div>
```

Comment:
You did a good job working on implementing the changes suggested by the previous reviewers. 👍 💯 

**_[OPTIONAL]_** - I could see that the hamburger is placed outside the `nav` tag. Please kindly consider wrapping the hamburger button inside the `nav` tag. The nav tag is for semantics reasons and user experience for mobile devices. 💪

Reply:
@abdulhamiid  thanks buddy I will do that, great looking out. I started following you on GitHub . keep up the good work